frontend/public/components/api-explorer.tsx

```diff
@@ -493,7 +494,10 @@ const APIResourceAccessReview: React.FC<APIResourceTabProps> = ({
   }, [apiGroup, apiVersion, plural, namespace, verb]);
 
   if (error) {
-    return (
+    // Don't love this fix to check for 403, but k8sCreate only returns error message ATM
+    return error.message.indexOf('is forbidden') > 0 ? (
```

Comment:
We shouldn't be inspecting the message response. I would rather leave the current behavior as-is (or always hide the try again button) if this is our only option.